helm_deploy/laa-estimate-eligibility/values/production.yaml

```diff
@@ -52,6 +52,6 @@ cfe:
   host: http://check-financial-eligibility-partner-check-financial-partner.check-financial-eligibility-partner-production.svc.cluster.local
 
 featureFlags:
-  sentry: NOT_ENABLED
+  sentry: ENABLED
   controlled: NOT_ENABLED
   asylumAndImmigration: NOT_ENABLED
```

Comment:
Can we turn emails off when sentry is enabled?

Reply:
@starswan we already do - https://github.com/ministryofjustice/laa-estimate-financial-eligibility-for-legal-aid/blob/main/config/environments/production.rb#L77